index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Animated Toolbar icons</title>
+    <script src="https://kit.fontawesome.com/e88d33689c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css"
+        href="https://cdn.jsdelivr.net/gh/loadingio/loading.css@v2.0.0/dist/loading.min.css" />
+    <link href="./style.css" rel="stylesheet" />
+</head>
+
+<body>
+    <header>
+        <div class="wrapper">
+            <nav>
+                <ul>
+                    <li class="calendar">
+                        <a href="#"><img src="./calendar-icon.gif"></a>
+                    </li>
+                    <li class="search">
+                        <a href="#"><img src="./search-icon.gif"></a>
+                    </li>
+                    <li class="event">
+                        <a href="#"><img src="./bolt-icon.gif"></a>
+                    </li>
+                    <li class="setting">
+                        <a href="#"><img src="./settings-icon.gif"></a>
+                    </li>
```

Comment:
A tip to help you reach the animation in the dribble:

- JS: For this challenge, you'll need a little bit of JS to add/remove a class ".active" to each `<li>`, when it's the active element. I also suggest you read a little about CSS nesting.
- Decompose the animations: This animation is made of multiple smaller animations. Some are simple, others are more complex. Divide them and start with the simpler one. For example, the little dot that appears in the bottom part when clicking one of the items.

Good luck!